### ISSUE TITLE
#encode(xml: :text) bug

### ISSUE THREAD

**Issue by h-nagoya** [hnagoya/use_salesforce_dataloader#3]
> `2000.encode(xml: text)`
> `NoMethodError: ...`
May be changed code `.encode` to `to_s.encode`.

**Comment by h-nagoya** [comment]
Like this?
```
case self
when String
    self.encode(xml: :text)
else
    self
end
```

**Comment by h-nagoya** [comment]
Fixed at 0.0.12.
